codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionHandlerTest.java

```diff
@@ -243,6 +245,17 @@ public void serverReceivingInvalidClientPrefaceStringShouldHandleException() thr
     }
 
     @Test
+    public void serverReceivingHttp1ClientPrefaceStringShouldIncludePreface() throws Exception {
+        when(connection.isServer()).thenReturn(true);
+        handler = newHandler();
+        handler.channelRead(ctx, copiedBuffer("GET /path HTTP/1.1", US_ASCII));
+        ArgumentCaptor<ByteBuf> captor = ArgumentCaptor.forClass(ByteBuf.class);
+        verify(frameWriter).writeGoAway(eq(ctx), eq(0), eq(PROTOCOL_ERROR.code()),
```

Comment:
you may not get the desired behavior by stacking `doAnswer` methods (not sure). What you have done lgtm and in the future is also a `reset` method which I believe dumps all mocked methods.